mlflow/tensorflow/__init__.py

```diff
@@ -138,21 +140,28 @@ def log_model(
     """
     Log a TF2 core model (inheriting tf.Module) or a Keras model in MLflow Model format.
 
+    .. note::
+
+        If logging model without signature, spark udf inference won't work if the model accepts
+        multidimensional tensors as inputs, and its pyfunc model cannot accept
+        pandas dataframe as inference inputs.
+        We can create signature by following code:
+
+        .. code-block:: python
+            :caption: Example of creating signature for saving TensorFlow and `tf.Keras` models
+
+            from mlflow.types.schema import Schema, TensorSpec
+            from mlflow.models.signature import ModelSignature
+            input_schema = Schema(
+                [
+                    TensorSpec(np.dtype(np.uint64), (-1, 5), "field1"),
+                    TensorSpec(np.dtype(np.float32), (-1, 3, 2), "field2"),
+                ]
+            )
+            signature = ModelSignature(inputs=input_schema)
```

Comment:
Proposal: Shall we provide a helper API for easier creating the signature ? The API could be like:

```
mlflow.tensorflow.create_model_signature([
  ("fieldA", [-1, 2, 3], np.float32),
  ("fieldB", [-1, 4, 3], np.float64),
])
```
@dbczumar @harupy

Reply:
Can we use `infer_signature`?